mercury_api/rpc/views.py

```diff
@@ -74,6 +91,10 @@ def get(self, job_id):
         :return: Job status dictionary. 
         """
         job = self.rpc_client.get_job_status(job_id)
+        for task in job['tasks']:
+            pass
```

Comment:
Is this for something to add later, or just to make sure 'tasks' exists and is iterable?

Reply:
yes, tasks will end up with links, too.

Reply:
adding wip tag